exasol/bucketfs/_path.py

```diff
@@ -272,6 +287,9 @@ def parent(self) -> str:
     def as_uri(self) -> str:
         return self._path.as_uri()
 
+    def as_udf_path(self) -> str:
+        return self._bucket_api.udf_path
```

Comment:
I think, you need to append `self._path` to the `self._bucket_api.udf_path`

Reply:
Yes, I do.